ansible/wordpress.yml

```diff
@@ -10,6 +10,13 @@
     wp_content_dir: "{{ wp_install_dir }}/wp-content"
   roles:
     - ansible-os-ubuntu-16
+    - role: cristian04.cloudwatch-monitoring
+    # CloudWatch basic monitoring uses 5 minute intervals. Enhanced monitoring supports up to 1 minute intervals, but results in extra charges.
+      cron_specs:
+        - name: "Cloudwatch monitoring"
+          minute: "*/5"
+          hour: "*"
+          job: "{{ cloud_watch_monitoring_path }}/aws-scripts-mon/mon-put-instance-data.pl --mem-util --mem-used --mem-avail --swap-util --swap-used --disk-space-util --disk-path={{ path }} --disk-space-avail --from-cron"
```

Comment:
Not a blocker: Maybe we put all this in a `monitoring_cron_specs` variable under `group_vars/all/` so that it can be shared everywhere?